pkgs/by-name/re/rex/package.nix

```diff
@@ -70,7 +70,7 @@ perlPackages.buildPerlPackage rec {
     FileLibMagic
     HashMerge
     HTTPMessage
-    IOPty
```

Comment:
Not sure if this should be changed. The rex source points to IO::Pty: https://github.com/RexOps/Rex/blob/master/t/os_dependencies.t#L14

Reply:
`IO::Pty` is included in the `IO-Tty` package so this seems fine?